cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4CareGapsProcessor.java

```diff
@@ -68,7 +70,8 @@ public Parameters getCareGapsReport(
             List<String> statuses,
             List<IdType> measureIds,
             List<String> measureIdentifiers,
-            List<CanonicalType> measureUrls) {
+            List<CanonicalType> measureUrls,
+            Boolean isDocumentMode) {
```

Comment:
Under what circumstances will a caller pass null instead of true or false?

Reply:
when a user is using old $care-gaps request where parameter is not specified

I have instead passed this as boolean to avoid